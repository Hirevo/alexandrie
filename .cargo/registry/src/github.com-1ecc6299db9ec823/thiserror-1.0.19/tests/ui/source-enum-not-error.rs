use thiserror::Error;

#[derive(Error, Debug)]
#[error("...")]
pub enum ErrorEnum {
    Broken {
        source: String,
    },
}

fn main() {}
