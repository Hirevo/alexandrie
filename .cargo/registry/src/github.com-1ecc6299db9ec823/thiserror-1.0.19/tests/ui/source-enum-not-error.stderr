error[E0599]: no method named `as_dyn_error` found for reference `&std::string::String` in the current scope
   --> $DIR/source-enum-not-error.rs:7:9
    |
7   |           source: String,
    |           ^^^^^^ method not found in `&std::string::String`
    |
    = note: the method `as_dyn_error` exists but the following trait bounds were not satisfied:
            `std::string::String: std::error::Error`
            which is required by `std::string::String: thiserror::aserror::AsDynError`
            `&std::string::String: std::error::Error`
            which is required by `&std::string::String: thiserror::aserror::AsDynError`
            `str: std::marker::Sized`
            which is required by `str: thiserror::aserror::AsDynError`
            `str: std::error::Error`
            which is required by `str: thiserror::aserror::AsDynError`
